tests/rewards/test_reward_nets.py

```diff
@@ -365,6 +529,19 @@ def test_serialize_identity(
         assert np.allclose(predictions[0], predictions[2])
 
 
+@pytest.mark.parametrize("env_name", IMAGE_ENVS)
+@pytest.mark.parametrize("net_cls", MAKE_IMAGE_REWARD_NET)
+@pytest.mark.parametrize("normalize_rewards", [True, False])
+def test_serialize_identity_images(
+    env_name,
+    net_cls,
+    normalize_rewards,
+    tmpdir,
+):
+    """Does output of deserialized reward CNN match that of original?"""
+    test_serialize_identity(env_name, net_cls, normalize_rewards, tmpdir)
```

Comment:
Bit weird that it's calling a test that is itself parameterized. I'm inclined to turn `test_serialize_identity` into a helper that then gets called by two parameterized tests.